### PR TITLE
Add border option for lsp_interop popup

### PR DESCRIPTION
**Pull request by ranjithshegde**
Just a small addition to optionally enable border for peek definition popup

### CODE REVIEW

File: doc/nvim-treesitter-textobjects.txt [hunk 1]
```diff
@@ -131,7 +131,9 @@ LSP interop~
   using Neovim's built-in LSP in a floating window. Press the shortcut twice
   to enter the floating window (when https://github.com/neovim/neovim/pull/12720
   or its successor is merged)
-
+Supported options:
+- enable: `true` or `false`.
+- border: 'none', 'single', 'double', 'rounded', 'solid', 'shadow'
```
Comment (stsewd) [inline doc/nvim-treesitter-textobjects.txt:136]:
```suggestion
- border: 'none' (default), 'single', 'double', 'rounded', 'solid', 'shadow'.
```

File: lua/nvim-treesitter-textobjects.lua [hunk 2]
```diff
@@ -52,6 +52,7 @@ function M.init()
       lsp_interop = {
         module_path = "nvim-treesitter.textobjects.lsp_interop",
         enable = false,
+		border = "none",
```
Comment (theHamsta) [inline lua/nvim-treesitter-textobjects.lua:55]: Can you have a closer look onto indentation here? It seems that tabs and spaces get mixed here

**Review by theHamsta — APPROVED** [review]

File: lua/nvim-treesitter-textobjects.lua (hunk 2 above)
Comment (ranjithshegde) [reply lua/nvim-treesitter-textobjects.lua:55]: Strange that it does that here. On my vim it is aligned with previous line. Will try changing my tabstop

**Review by theHamsta — APPROVED** [review]

**Comment by ranjithshegde** [comment]
woops! merged before the intent was fixed!

**Pull request merged**